app/models/provider.rb

```diff
@@ -47,12 +48,12 @@ class Provider < ApplicationRecord
            class_name: "ProviderEnrichment"
   has_many :courses
 
-  scope :changed_since, ->(datetime) do
-    if datetime.present?
-      where("last_published_at >= ?", datetime)
+  scope :changed_since, ->(timestamp) do
+    if timestamp.present?
+      where("provider.changed_at > ?", timestamp)
     else
-      where("last_published_at is not null")
-    end.order(:last_published_at, :id)
+      where("changed_at is not null")
+    end.order(:changed_at, :id)
```

Comment:
`end.order` just looks odd

Reply:
It's called functional programming. :) Do some lisp and it'll start to feel more comfortable.

Having said that, I'm a little reluctant to introduce an intermediary variable here, unless you think it would really help out? For example:

```
where_clause = if timestamp.present?
                 where("provider.changed_at > ?", timestamp)
               else
                 where("changed_at is not null")
               end
where_clause.order(:changed_at, :id)
```

Reply:
i was just think brackets...

vs reading out loud as `end order`

Reply:
I'd keep the `end.order` if it's idiomatic rails, I could get used to it